processes/release_processes/firecloud-develop.dot

```diff
@@ -42,7 +42,6 @@ digraph {
     dspjenkins_merge [shape=oval label="Merge dsp-jenkins PR"];
 
     jenkins_set [shape=oval label="[Jenkins] Use dsl-seed to make our dsp-jenkins branch the default"];
-    jenkins_reset [shape=oval label="[Jenkins] Use dsl-seed to make 'master' the default again"];
```

Comment:
This step no longer appears to be necessary and apparently was recently removed from the DAG but not the DOT file so it was confusingly rendered as a dissociated bubble with no arrows in or out.

Reply:
🤔  I hope that was an intentional change.

Reply:
Yes the `dsl-seed` job seems to run automatically to reset to `master` when the branch in `dsp-jenkins` is merged.